form.php

```diff
@@ -4,7 +4,12 @@
 
 $db = connectToDB();
 
-if (count($_POST) === 5) {
+if (!empty($_POST['delete_title'])) {
+    updateDB($db, $_POST['delete_title']);
+    header('Location: index.php');
+}
+
+if (!empty($_POST['title'] && $_POST['author'] && $_POST['genre'] && $_POST['released'] && $_POST['page_count'])) {
```

Comment:
I recall mike saying if there was more then 2 or 3 conditionals maybe to put it in its own function